### DISCUSSION TITLE
Add hyphens utility

### DISCUSSION THREAD

**Discussion by karlerikjonatan** [tailwindlabs/tailwindcss#7286] [Ideas]
Hi 👋

I was doing some typography work today and noticed the lack of a hyphens utility. The hyphens CSS property specifies how words should be hyphenated when text wraps across multiple lines.

Would there be of any interest adding a utility as such? If so, I would love to give back to the Tailwind CSS community and contribute with a PR solving this issue.

![hyphens-none](https://user-images.githubusercontent.com/2832719/151994657-f76934d6-ca98-4725-8b17-8c074e7fd3f1.png)
![hyphens-auto](https://user-images.githubusercontent.com/2832719/151994702-29e8fae3-790d-4cbb-a4f7-e51b68a0a215.png)

**Comment by module17** [comment]
I think it is a good addition. I was quite surprised to see that it is missing.

**Comment by fratzinger** [comment]
I've created a PR: https://github.com/tailwindlabs/tailwindcss/pull/10071

**Comment by karlerikjonatan** [comment]
https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.3.0
